Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -545,3 +545,9 @@
 "status-saved-for-later" = "Saved for later";
 
 "announcements-dismiss" = "No thanks";
+
+"places-search-top-articles" = "Top articles";
+"places-search-search-mapped-area" = "Search mapped area";
```

Comment:
After design review this was changed to "search this area" - i also removed mapped from the key